be/test/vec/columns/column_object_test.cpp

```diff
@@ -160,11 +162,11 @@ TEST_F(ColumnObjectTest, test_insert_indices_from) {
 
         Field result1;
         dst_column->get(0, result1);
-        EXPECT_EQ(result1.get<VariantMap>().at("").get<Int64>(), 123);
+        EXPECT_EQ(result1.get<VariantMap>().at({}).get<Int64>(), 123);
```

Comment:
When we get the value from VariantMap, do we always need to execute `.at({})` first?

Reply:
calling insert_from

Reply:
execute `.at({})` only when access root column